README.md

```diff
@@ -1,4 +1,7 @@
-# jsdom
+<h1 align="center">
+    <a href="https://github.com/jsdom/jsdom"><img width="200" height="200" src="logo.svg" alt="jsdom" /></a><br />
```

Comment:
Nit: This seems like an appropriate case for using a blank alt-attribute (`alt=""`), since it only repeats the title that follows. You can also remove the `/` from the `<img>` and `<br>` elements.

Reply:
Self-closing slash is optional in HTML5 doctype. But I agree with [this](http://xahlee.info/js/html5_non-closing_tag.html).  
Also, [`parse5` example](https://raw.githubusercontent.com/inikulin/parse5/master/README.md) has it. Do you still want to remove them?

Reply:
I remember that I made similar arguments for myself back when HTML5 first started gaining traction, since I'd been writing XHTML (or the "XML syntax") for years. It didn't take me too long to get used to the differences though.

We tend to omit the optional parts of HTML for most code in this repo, so in this case it is mainly about being consistent.